docfx.json

```
@@ -192,7 +192,8 @@
         "docs/standard/design-guidelines/*.md": "dotnet",
         "docs/standard/security/*.md": "dotnet",
         "docs/visual-basic/**/**.md": "dotnet-visualbasic",
-        "docs/spark/**/**.md": "dotnet-spark"
+        "docs/spark/**/**.md": "dotnet-spark",
+        "includes/**/**.d": "dotnet"
```

Typo?

```suggestion
        "includes/**/**.md": "dotnet"
```

And does it actually have an effect?